### PR TITLE
Remove .NET 6 Targets (End of Life)

### PR DESCRIPTION
**Pull request by inforithmics**
Removed .Net 6 Targets because they are end of life as of November 2024 (And could cause problems later on).

### CODE REVIEW

File: Mapsui.Nts/Mapsui.Nts.csproj.Backup.tmp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (pauldendulk) [inline Mapsui.Nts/Mapsui.Nts.csproj.Backup.tmp:1]: I think this was added by accident.

**Review by pauldendulk — COMMENTED** [review]
Great! Please look at my remark.

File: Mapsui.Nts/Mapsui.Nts.csproj.Backup.tmp (hunk 1 above)
Comment (inforithmics) [reply Mapsui.Nts/Mapsui.Nts.csproj.Backup.tmp:1]: Yes you are right

**Comment by inforithmics** [comment]
Is now fixed

**Review by pauldendulk — APPROVED** [review]

**Pull request merged**